app/services/group_action_logger.rb

```diff
@@ -21,17 +21,19 @@ def log_remove_user_as_group_owner(target_user)
     ))
   end
 
-  def log_add_user_to_group(target_user)
+  def log_add_user_to_group(target_user, subject = nil)
```

Comment:
@davidtaylorhq Just a note that I've revmoved those added methods (which seems to have deleted your comment) and added an additional param to the existing methods.